### DISCUSSION TITLE
Sigma File Manager v1.3.0

### DISCUSSION THREAD

**Discussion by aleksey-hoffman** [aleksey-hoffman/sigma-file-manager#130] [Announcements]
# Release Notes:

This release improves stability, performance, and also brings a few new features:

- 1 new major feature
- 2 new minor features
- 12 other improvements
- 4 fixes

## Milestones:

💗 **(Goal: 2 of 30 individuals & 0 of 10 companies supporters)** 
New supporter joined our community on Patreon, welcome:

<div>
  <a href="https://github.com/andyundso" title="Andy Pfister">
    <b>~ Andy Pfister ~</b>
  </a>
  web developer
</div>

## New major features:

- **Feature**: Customizable window transparency effect | e54418a

## New minor features:

- **Feature**: add new shortcuts: `addDirItemsForCopying`, `addDirItemsForMoving`, which allow you to copy / move items from multiple 
 different directories at once | 5ef4a9a
- **Feature**: add ability to create `symlinks`, `hard links` and Windows `.lnk shortcuts` from the context menu by pressing different modifiers | 478b3dc

## Other improvements:

- **UI**: added new artwork for home banner: "Ice Cave by Wang Jie" | 2d40d4c
- **UI**: added new artwork for home banner: "Slum by Vladimir Manyukhin" | cdbfefd
- **UX**: the current directory will now reload immediately after deleting / trashing items | 17cceba
- **UI**: notification cards will now have a nicer and more consistent design | 65b9a41
- **UI**: items in the range selection will now be highlighted immediately when `Shift` key is pressed | 0ca8ff2
- **UI**: filter field menu design was improved | 2a6ef8f
- **UX**: you can now specify a custom path when creating a new link / shortcut  | dc7ae34
- **UX**: icons in context-menu will now dynamically change when you press different key modifiers, and their tooltips will look nicer and more consistent | 106191f
- **UX (for developers)**: All copied paths will now be automatically wrapped with quotes. On Windows, selected paths will now be copied with single backward `\` slashes, or with double backward `\\` slashes by pressing a key modifier | 86695f2
- **UX**: `zoomIncrease` shortcut will now also be triggered with `Ctrl + Plus` on the numpad | 4f90117
- **Dev**: Code optimizations and refactoring
- A few other small improvements in UI

## Fixed:

- **UX**: fixed the issue with archiver feature not working on Linux and MacOS because of bin permissions | 1d646ca
- **UI**: the animation of navigation panel will now work correctly on home page | 24ebe35
- **UI**: removed excessive bottom padding when the bottom clipboard toolbar is shown | d78d7e8
- **UX**: fixed the issue with items getting added to copy / move clipboard when you select other items | 51f345a

## Supporters

![Patreon-progress-goal-1](https://user-images.githubusercontent.com/61761672/129417221-95d3c176-54ff-4c53-a285-adcb974c8254.png)

When we meet the goal, I will be able to dedicate more of my time on the app development, and add new features more frequently.

This release was made possible by my Patreon supporters.
Show them some love, check out their work:

<div>
  <a href="https://www.photoancestry.com/" title="Houston Photo Restoration">
    <b>~ Houston Photo Restoration</b>
  </a>
  <a href="https://github.com/andyundso" title="Andy Pfister">
    <b>~ Andy Pfister ~</b>
  </a>
</div>

#### 🎉 Supporter goal: 2 of 30 people joined

Supporters get rewards for supporting this project on Patreon:

- Your issues (feature requests) will have higher priority and a `supporter 💖` label;
- You will be able to vote and decide which features should be prioritized;
- You will get to see Patreon-only posts, and screenshots showcasing new features during development;
- Your name and the link to your project / website will be displayed on this page;
- You will get access to a private Discord where you can learn, get advice, get help, have fun, and just chat with me and other community members;
- Also, thanks to your support, you and everyone else will be getting new features in the app more frequently.

If you like the app, consider supporting my work by joining our community on Patreon. I have a lot of awesome things in development for you guys, but the luck of funding makes it really difficult to create anything.

Check out the full list of rewards on Patreon:

<a target="_blank" href="https://patreon.com/sigma_file_manager">
  <img
    src="https://github.com/aleksey-hoffman/sigma-file-manager/blob/main/.github/media/patreon_button.png"
    width="164px"
  />
</a>

## Download

You can find the download link for your system in the **Asset** section below.

## Notes

- Windows OS is currently the primary target platform, hence MacOS and Linux builds have more problems and some of the features don't work as expected. The issues will be gradually fixed in future updates.
- MacOS build doesn't meet my quality standards yet, it has too many problems, I decided to not publish it until the next update in a few days.

## Installation notes

This is a new app and it doesn't have a signed certificate yet, so you might see a security warning on Windows and MacOS during installation.
- On Windows 10: if you see the Smart Screen prompt, click **"More Info"** and then **"Run Anyway"**. 

<hr /><em>This discussion was created from the release <a href='https://github.com/aleksey-hoffman/sigma-file-manager/releases/tag/v1.3.0'>Sigma File Manager v1.3.0</a>.</em>

**Comment by Ansh4011** [comment]
Thanks!
